lib/components/molecules/StickyHeader/StickyHeader.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useRef, useEffect, Fragment } from 'react';
+import type, { Node } from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+import styles from './StickyHeader.style';
+import './StickyHeader.css';
+
+type Props = {
+  children: Node,
```

Comment:
Define className too, and both should be at type/index.js

Reply:
fixed.